package.json

```diff
@@ -87,14 +87,14 @@
         }
     },
     "dependencies": {
-        "async": "1.5.2",
+        "async": "2.6.1",
         "colors": "1.3.3",
-        "github": "~0.2.4",
+        "github": "~14.0.0",
```

Comment:
you really should update to @octokit/rest instead :) There are plenty of breaking changes between v16 and v15, but I think the the release notes help you through the process.

Love this lib, will definitely check it out!

Reply:
Thanks for the comment @gr2m and for all you do! I am in the process of un-nesting callback hell into using async await with @octokit/rest and it is a bit of an upgrade haha.  Then will be looking forward to leveraging @octikit/graphql.js where it makes sense  and possibly @octokit/octokit.js in the future 🎱